tests/unit/test-util/test.adb.js

```diff
@@ -49,6 +50,13 @@ android.permission.READ_EXTERNAL_STORAGE, granted=true
 android.permission.WRITE_EXTERNAL_STORAGE, granted=true
 `;
 
+const file = {
+  name: artifactDir,
+  isDirectory: () => {
+    return true;
+  },
+};
```

Comment:
Same here, we could create the fake file objects into the related test case.
Especially because we should test the helper function on mode than just one file that is a directory (we should also be sure that the behavior will be the expected one when there are files that are not directories or directories that do not match the expected prefix in their name)